client/src/components/ProgressWheel/ProgressWheel.module.css

```diff
@@ -0,0 +1,31 @@
+.svg-box {
+  border-radius: 50%;
+}
+
+.circle-background,
+.circle-progress {
+  fill: none;
+}
+
+.circle-background {
+  stroke: #ddd;
+}
+
+.circle-progress {
+  stroke: #f15852;
```

Comment:
can we use CSS variables here?

Reply:
This is something to be done by the UX/UI people !

Reply:
@Joaoviana is it worth making an issue so that it doesn't get overlooked?